Update app-service-web-get-started-python.md

temores opened the pull request:
Update bash code as from cd Python-docs-hello-world to cd python-docs-hello-world, the example in the website will fail as folder with capital P does not exist.

azurecla commented:
Hi __@itaysk__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

itaysk commented:
MS FTE - itays

Thraka commented:
I've notified the original author of your requested change and will wait for his feedback. Thanks!

temores committed:
Update app-service-web-get-started-python.md

Update bash code as from cd Python-docs-hello-world to cd python-docs-hello-world, the example in the website will fail as folder with capital P does not exist.
